chatterbot/trainers.py

```diff
@@ -78,7 +78,7 @@ def train(self, conversation):
         """
         statement_history = []
 
-        for text in conversation:
+        for text in reversed(conversation):
```

Comment:
@andpark Thanks for contributing, The train reverse will not work for all. Instead you could make hack like this, If command line argument has train method you could reverse.

``` Python
if 'train' in sys.argv:
    conversation = reversed(conversation)
```

however this is not a good method, instead you could try a better solution for example detect Django command type 

for example
``` Python
if django.command.type == "train"
    "do your operations"
```